Assets/Scripts/Framework/puzzels/SequencePuzzle/SequenceButton.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+public class SequenceButton : SequenceItem<EnumColor> 
+{
+    public EnumColor ButtonValue;
```

Comment:
```suggestion
    [SerializeField] private EnumColor value;
```
kan private of nee ik zie nergens in andere scripts